### ISSUE TITLE
Types for V3 webhooks

### ISSUE THREAD

**Issue by KazimirPodolski** [nylas/nylas-nodejs#580]
**Is your feature request related to a problem? Please describe.**
V2 had webhooks types like WebhookNotification and whatnot.

**Describe the solution you'd like**
It makes sense to have them in V3 API (V7 SDK).

**Comment by juan-talin** [comment]
Yeah I'd need a bump on this please. Really thought that V3 SDK would have V2 parity.

**Comment by AaronDDM** [comment]
Sorry for the delay, I shall take a look at this. I am assuming however this is related to Webhook Triggers not being exported.
